platform/security/sts/security-sts-realm/src/main/java/ddf/security/realm/sts/StsRealm.java

```diff
@@ -821,4 +854,179 @@ public boolean doCredentialsMatch(AuthenticationToken token, AuthenticationInfo
         }
     }
 
+    @Override
+    public String getAddress() {
+        return address;
+    }
+
+    @Override
+    public void setAddress(String address) {
+        this.address = address;
+    }
+
+    @Override
+    public String getEndpointName() {
+        return endpointName;
+    }
+
+    @Override
+    public void setEndpointName(String endpointName) {
+        this.endpointName = endpointName;
+    }
+
+    @Override
+    public String getServiceName() {
+        return serviceName;
+    }
+
+    @Override
+    public void setServiceName(String serviceName) {
+        this.serviceName = serviceName;
+    }
+
+    @Override
+    public String getUsername() {
+        return username;
+    }
+
+    @Override
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    @Override
+    public String getPassword() {
+        return password;
+    }
+
+    @Override
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    @Override
+    public String getSignatureUsername() {
+        return signatureUsername;
+    }
+
+    @Override
+    public void setSignatureUsername(String signatureUsername) {
+        this.signatureUsername = signatureUsername;
+    }
+
+    @Override
+    public String getSignatureProperties() {
+        return signatureProperties;
+    }
+
+    @Override
+    public void setSignatureProperties(String signatureProperties) {
+        this.signatureProperties = signatureProperties;
+
+    }
+
+    @Override
+    public String getEncryptionUsername() {
+        return encryptionUsername;
+    }
+
+    @Override
+    public void setEncryptionUsername(String encryptionUsername) {
+        this.encryptionUsername = encryptionUsername;
+
+    }
+
+    @Override
+    public String getEncryptionProperties() {
+        return encryptionProperties;
+    }
+
+    @Override
+    public void setEncryptionProperties(String encryptionProperties) {
+        this.encryptionProperties = encryptionProperties;
+
+    }
+
+    @Override
+    public String getTokenUsername() {
+        return tokenUsername;
+    }
+
+    @Override
+    public void setTokenUsername(String tokenUsername) {
+        this.tokenUsername = tokenUsername;
+    }
+
+    @Override
+    public String getTokenProperties() {
+        return tokenProperties;
+    }
+
+    @Override
+    public void setTokenProperties(String tokenProperties) {
+        this.tokenProperties = tokenProperties;
+    }
+
+    @Override
+    public List<String> getClaims() {
```

Comment:
Access to the `claims` field should probably be protected by lock/sync semantics and this method should probably return a copy or `unmodifiableList`.